src/rocksdb.rs

```diff
@@ -468,31 +421,15 @@ impl DB {
         self.write_opt(batch, &wo)
     }
 
-    pub fn get_opt(&self,
-                   key: &[u8],
-                   readopts: &ReadOptions)
-                   -> Result<Option<DBVector>, Error> {
+    pub fn get_opt(&self, key: &[u8], readopts: &ReadOptions) -> Result<Option<DBVector>, Error> {
         if readopts.inner.is_null() {
-            return Err(Error::new("Unable to create rocksdb read options.  \
-                                   This is a fairly trivial call, and its \
-                                   failure may be indicative of a \
-                                   mis-compiled or mis-loaded rocksdb \
-                                   library."
-                .to_string()));
+            return Err(Error::new("Unable to create rocksdb read options. This is a fairly trivial call, and its failure may be indicative of a mis-compiled or mis-loaded rocksdb library.".to_owned()));
```

Comment:
This line is a bit of an eye-bleeder for me. I read text formatted this way much more slowly than narrow columns, and I have to hit more keys in my editor to move the window around to read it all. My eyes and fingers would appreciate it if you didn't join lines so aggressively.